osu.Game/Graphics/UserInterface/KeyCounterMouse.cs

```diff
@@ -15,18 +18,25 @@ public KeyCounterMouse(string name, MouseButton button) : base(name)
         {
             Button = button;
         }
+        private Bindable<bool> mouseButtonsDisabled;
+
+        [BackgroundDependencyLoader]
+        private void load(OsuConfigManager config)
+        {
+            mouseButtonsDisabled = config.GetBindable<bool>(OsuConfig.MouseDisableButtons);

```

Comment:
I don't like that check against the mouse disabled state in the key counter. I feel like the only place we should check that is inside `PlayerInputManager` and `KeyCounterMouse` should simply be inside `PlayerInputManager` so that it never gets mouse down events in the first place if they are disabled.